src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/InstancesController.cs

```diff
@@ -424,8 +430,14 @@ public class InstancesController : ControllerBase
             {
                 // start process and goto next task
                 instanceTemplate.Process = null;
-                string startEvent = await _altinnApp.OnInstantiateGetStartEvent();
-                processResult = _processService.ProcessStartAndGotoNextTask(instanceTemplate, startEvent, User);
+
+                ProcessChangeContext processChangeContext = new ProcessChangeContext();
```

Comment:
I assume some of the properties being set below are required for the ProcessChangeContext to be in a valid state, if so then I think we should add a constructor with the minimum required properties and only set the optional ones via properties.

Reply:
Added new constructor

Reply:
Maybe I'm missing something here, but did you use the new constructor? I still only see the use of default empty constructor with property assignments.

Reply:
Yes. I use the new constructor.